code_samples/live_templates/src/main/resources/META-INF/plugin.xml

```diff
@@ -34,8 +34,12 @@
   <vendor url="https://plugins.jetbrains.com">IntelliJ Platform SDK</vendor>
 
   <extensions defaultExtensionNs="com.intellij">
+    <!-- Declare where the IntelliJ Platform should look to find XML-based Live Templates   -->
```

Comment:
I'm not in favor of having so many comments for "plain" extension points like these in plugin.xml

Reply:
Ok.